go-controller/pkg/node/management-port_linux.go

```diff
@@ -220,6 +220,27 @@ func setupManagementPortIPFamilyConfig(mpcfg *managementPortConfig, cfg *managem
 		return warnings, err
 	}
 
+	createForwardingRule := func(family string) error {
```

Comment:
lol not sure if this is the right place to do this, for some reason this code is being run every 30 seconds...

Reply:
I guess this is the main func where we create the mp0 interface and related things.. the 30 second poll for redoing all this is happening because of:
```
func (mp *managementPortNetdev) CheckManagementPortHealth(cfg *managementPortConfig, stopChan chan struct{}) {
	go wait.Until(
		func() {
			checkManagementPortHealth(cfg)
		},
		30*time.Second,
		stopChan)
}
```
and I don't think we need to retry setup mp0 every 30 seconds?

Reply:
we do need to retry it in case the port is deleted by a user or NM or something like that. I think where you put the code is fine.

Reply:
ack. leaving it as is then.